src/common/components/contactInfo/contactInfo.component.js

```diff
@@ -41,6 +41,7 @@ class Step1Controller {
     }
 
     this.loadDonorDetails(donorDetailsDefaults)
+    this.loadRadioStations()
```

Comment:
This commit (13592e625187459d15f769d4e5e85cc2d5753449) should be squashed with 1282eedaacdd82fa9f941146ded196bc91055d24 and given a nice name about contact info being done